pg-extend/src/lib.rs

```diff
@@ -85,19 +87,28 @@ pub fn register_panic_handler() {
     panic::set_hook(Box::new(|info| {
         let level = pg_error::Level::Fatal;
 
-        pg_error::log(level, file!(), line!(), module_path!(), format!("panic in Rust extension: {}", info));
+        // TODO: downcast info, check if it's the value we need.
```

Comment:
here we'll check the panic data, and if it was due to a longjmp, we'll rethrow the longjmp...